src/14_cal.py

```diff
@@ -21,4 +21,34 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+current_year = datetime.now().year
+current_month = datetime.now().month
+
+user_input = input(
+    'Enter month and year (optional) separated by a space: ').split(' ')
+
+
+def generate_calendar(ui):
+    if len(ui) == 1 and ui[0] == '':
+        print(calendar.month(current_year, current_month))
+    elif len(ui) == 1:
+        try:
+            monthInt = int(ui[0])
+            print(calendar.month(2019, monthInt))
+        except ValueError:
+            print('Only numbers can be entered.')
+
```

Comment:
I'm impressed. Day two of Python and you're validating user input with a try, except... phenomenal! Love the `ui == user input` here.